src/Components/Facet.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import {
+  Facet as FacetType,
+  FacetState,
+  buildFacet,
+  FacetValue
+} from "@coveo/headless";
+import { headlessEngine } from "../Engine";
+import FormLabel from "@material-ui/core/FormLabel";
+import FormControl from "@material-ui/core/FormControl";
+import FormGroup from "@material-ui/core/FormGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Button from "@material-ui/core/Button";
+import Checkbox from "@material-ui/core/Checkbox";
+
+interface IFacetProps {
+  title: string;
+}
+
+export default class Facet extends React.Component<IFacetProps, {}> {
+  private headlessFacet: FacetType;
+  state: FacetState;
+
+  constructor(props: any) {
+    super(props);
+
+    this.headlessFacet = buildFacet(headlessEngine, {
+      options: {
+        numberOfValues: 3,
```

Comment:
I think the field should be passed as a prop instead of being hardcoded here.

Reply:
Let me know if you need some help with this @jgeddes-coveo 👍 
After this I think we can add people from ui-kit to the review.

Reply:
This is the suggestion that I was waiting on before re-reviewing @jgeddes-coveo :) 
Ping me if you'd like to have a call to fix it.

Reply:
This is the change that Kaitlyn and I did following Fred's suggestion, @jpmarceau. Do you see a problem with it?

Reply:
hmmm... I doubt that's what Fred meant 🤔 
on the next line, you can see that `field` is passed as a prop. That's what you'd want to do with `numberOfValues` too.

Reply:
oups my bad I just got confused because of the meanings of `field` 😅